apis/appmesh/v1beta2/virtualnode_types.go

```diff
@@ -331,6 +331,9 @@ type AWSCloudMapServiceDiscovery struct {
 type DNSServiceDiscovery struct {
 	// Specifies the DNS service discovery hostname for the virtual node.
 	Hostname string `json:"hostname"`
+	// Choose between ENDPOINTS (strict DNS) and LOADBALANCER (logical DNS) mode in Envoy sidecar
+	// +optional
+	ResponseType *string `json:"responseType,omitempty"`
```

Comment:
Can we add this validation here ? 
// +kubebuilder:validation:Enum=ENDPOINTS;LOADBALANCER;

Reply:
++ added!